db/migrate/20191101055132_backfill_granted_title_badge_id_for_user_profile.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class BackfillGrantedTitleBadgeIdForUserProfile < ActiveRecord::Migration[6.0]
+  def up
+    # update all the regular badge derived titles based
+    # on the normal badge name
+    ActiveRecord::Base.connection.execute <<-SQL
+      UPDATE user_profiles
+      SET granted_title_badge_id = b.id
+      FROM users
+      INNER JOIN badges b ON users.title = b.name
+      WHERE users.title = b.name
```

Comment:
Why we checking the `users.title = b.name` both in inner join condition and where condition?